tests/integration/test_remoted/conftest.py

```diff
@@ -1,15 +1,17 @@
 # Copyright (C) 2015-2021, Wazuh Inc.
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import logging
+import os
+import subprocess
 import pytest
-
+import subprocess as sb
```

Comment:
This import is duplicated in line 6

Reply:
https://github.com/wazuh/wazuh-qa/pull/1115/commits/3188d703e6859966b0ef6777feccd899cf435f59